package.json

```diff
@@ -11,8 +11,9 @@
   "engines": {
     "node": ">=10"
   },
+  "types": "dist/index.d.ts",
   "scripts": {
-    "build": "microbundle-crl --no-compress --format modern,cjs",
+    "build": "microbundle-crl --no-compress --format modern,cjs && cp src/index.d.ts dist",
```

Comment:
@mifi - I know you'll be publishing this manually - do you have a Windows or MacOS machine? `cp` will only work on windows, so we may need to look into a dev pkg that can execute `copy` on win & MacOS

Reply:
I'm not sure why cp will only work on windows. MacOs has the cp command too. I use macos so it should be ok, we can improve this in the future if needed. pr welcome

Reply:
sorry, my apologies a typo- the `cp` command will only work on linux / unix.